hello/bin/hello.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "../src/hello.h"
+
+int
+main(int argc, char **argv) {
```

Comment:
Just habitually.. I have no important reason why "bin".
Do you know any good idea?